README.md

```diff
@@ -2740,3 +2743,233 @@ Web APIs are commonly used for various purposes, such as accessing data from ext
 </div>
 
 ---
+
+<a name="how-does-destructuring-work-in-javascript-and-what-are-its-use-cases?"></a>
+
+## How does destructuring work in javascript, and what are its use cases?
```

Comment:
@travisliu plase add the number in front of the title in each section as all the others.